### DISCUSSION TITLE
是否有方法只允许本机访问代理?

### DISCUSSION THREAD

**Discussion by kaseeen** [v2rayA/v2rayA#1439] [Q&A]
个人测试 开启端口分享后不仅局域网可以访问 公网同时也可以访问端口 
请问是否有方法做到只允许局域网或者本机访问代理端口?

**Comment by eronez** [comment]
1. firewall
2. hook config https://v2raya.org/docs/advanced-application/custom-extra-config/

**Reply by eronez** [reply]
for debian series,change /usr/lib/systemd/system/v2raya.service 
`ExecStart=/usr/bin/v2raya -a 127.0.0.1:2017 --log-disable-timestamp`

**Comment by M0rtale** [comment]
`sudo iptables -A INPUT -p tcp --dport 2017 -j DROP`

之后直接用 localhost:2017 来访问
